concent_api/core/tests/test_unit_model.py

```diff
@@ -162,3 +170,243 @@ def test_that_storing_subtask_with_task_to_compute_nested_in_another_messages_wi
                 force_get_task_result=force_get_task_result,
                 subtask_results_rejected=subtask_results_rejected,
             )
+
+
+class TestDepositAccountValidation(ConcentIntegrationTestCase):
+    (PROVIDER_PRIVATE_KEY, PROVIDER_PUBLIC_KEY) = generate_ecc_key_pair()
+
+    def generate_ethereum_public_key(self):  # pylint: disable=no-self-use
+        ecc = ECCx(None)
```

Comment:
Please use ethereum public key from `ConcentIntegrationTestCase` test class, ie. `self.PROVIDER_PUB_ETH_KEY`.

Reply:
Done.

Reply:
The old way is still used. Please use existing key instead of generating new one.